src/main/java/mate/academy/model/CinemaHall.java

```diff
@@ -0,0 +1,71 @@
+package mate.academy.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.util.Objects;
+
+@Entity
+@Table(name = "CinemaHalls")
+public class CinemaHall {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private int capacity;
+    private String description;
+
+    public Long getId() {
+        return id;
+    }
+
+    public CinemaHall setId(Long id) {
+        this.id = id;
+        return this;
+    }
```

Comment:
The `setId` method might not be necessary since the ID is auto-generated. Consider removing this setter to prevent accidental changes to the ID.